.github/workflows/smoke-test-ipv6.yaml

```diff
@@ -57,14 +57,14 @@ jobs:
         run: |
           helm upgrade -i cilium ./install/kubernetes/cilium \
             --namespace kube-system \
-            --set global.nodeinit.enabled=true \
-            --set global.kubeProxyReplacement=strict \
+            --setnodeinit.enabled=true \
+            --setkubeProxyReplacement=strict \
```

Comment:
Not having a space between `set` and `nodeinit.enabled=true` is allowed in helm? Can you please add a space. It's very hard to read otherwise :cry:

Reply:
Totally by accident, thanks for catching it.